### ISSUE TITLE
`Error: read-only image store assigns the same name to multiple images`

### ISSUE THREAD

**Issue by wahabk** [NERSC/podman-hpc#119]
Hello! We have ran into this issue:

```shell
$ podman-hpc images
Error: read-only image store assigns the same name to multiple images
$ podman-hpc image ls
REPOSITORY                            TAG         IMAGE ID      CREATED      SIZE
<IMAGE_NAME>  v1.3        7589b976db7f  2 weeks ago  15 GB
$ podman-hpc images
Error: read-only image store assigns the same name to multiple images
$ podman-hpc ps
CONTAINER ID  IMAGE       COMMAND     CREATED     STATUS      PORTS       NAMES
[no running containers]
```

The only similar issue I could find is:

https://bugzilla.redhat.com/show_bug.cgi?id=1952637

I checked the `images.json` and `layers.json` on the login and compute nodes and I found no duplicates. We had to remove the entire `overlay*` directories and start from scratch.

Have you ran into this before? Or know of methods to debug this?

**Comment by scanon** [comment]
Yes.  I think we have a fix for this.  The issue I believe was collisions in the name history field that wasn't being managed.  I need to check on the status of that fix.

**Comment by moustakas** [comment]
In case it's helpful, I'm also seeing the same issue intermittently---but not reproducibly.

**Comment by scanon** [comment]
I believe this is fixed in the latest version.  I will close unless there is a new reproducer.
